src/HealthChecks/HealthChecks/src/DependencyInjection/HealthChecksBuilderAddCheckExtensions.cs

```diff
@@ -48,7 +48,9 @@ public static class HealthChecksBuilderAddCheckExtensions
         /// <param name="tags">A list of tags that can be used to filter health checks.</param>
         /// <param name="timeout">An optional <see cref="TimeSpan"/> representing the timeout of the check.</param>
         /// <returns>The <see cref="IHealthChecksBuilder"/>.</returns>
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
@sharwell recommended using the following in his review of #24188:
``` msbuild
[SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "Required to maintain compatibility")]
```
See https://github.com/dotnet/aspnetcore/pull/24188#discussion_r458957296 but this also reduces the noise to a single line😺